aws-ts-ruby-on-rails/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "aws-ec2-rails",
+    "version": "0.1.0",
+    "main": "bin/index.js",
+    "typings": "bin/index.d.ts",
+    "scripts": {
+        "build": "tsc"
+    },
+    "devDependencies": {
+        "typescript": "^2.7.2"
+    },
+    "dependencies": {
+        "@pulumi/aws": "^0.14.0",
+        "@pulumi/pulumi": "^0.14.0",
+        "@types/mustache": "^0.8.31",
```

Comment:
Technically @types can be devDependencies.

Reply:
Good catch, will fix.